### DISCUSSION TITLE
Confirmation Link and NextJS Routes

### DISCUSSION THREAD

**Discussion by JSsantana** [supabase/supabase#2145] [Questions] [answered]
NextJS route use pages folder to routing. When I activate the account using the confirmation URL provided by supabase I get an error page and I don't have idea how deal with this. Can you please give me an example how to show a success page with the supabase confirmation link?

**Comment by inian** [comment, marked as answer]
Hi @JSsantana, we have a full Next.js guide which uses Supabase Auth [here](https://supabase.io/docs/guides/with-nextjs#set-up-a-login-component). 

You can also consider using our [Auth component](https://ui.supabase.io/components/auth) from our Supabase UI library
